update.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [[ -z "$1" ]]; then
+    echo "Usage: bash update.sh version" 1>&2
```

Comment:
I put replaceable arguments in angle brackets (`<version>`) but that's a personal style.  Up to you either way.

Reply:
That's a good call. Added.